spec/datetimeformat.html

```diff
@@ -303,6 +304,10 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"relativeYear"`, then
```

Comment:
Should this be relativeYear or relatedYear ?

Reply:
Oops! Fixed.